docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+
+networks:
+  mh-network:
+    driver: bridge
+
+services:
+  anisette:
+    image: dadoum/anisette-v3-server
+    container_name: anisette
+    restart: always
+    ports:
+      - "6969:6969"
+    volumes:
+      - anisette-v3_data:/home/Alcoholic/.config/anisette-v3/lib/
```

Comment:
Make it:
```
anisette-v3_data:/home/Alcoholic/.config/anisette-v3
```
So that It persists the device and all relevant data between ups and downs... if not you will need to re-login on each `docker compose up`... 🤷‍♂️

Reply:
Fair point, fixed it.